src/praisonai-agents/praisonaiagents/mcp/mcp_http_streaming.py

```
@@ -0,0 +1,353 @@
+"""
+HTTP-Streaming client implementation for MCP (Model Context Protocol).
+This module provides the necessary classes and functions to connect to an MCP server
+over HTTP using chunked transfer encoding for bidirectional streaming.
+"""
+
+import asyncio
+import json
+import logging
+import threading
+import inspect
+from typing import List, Dict, Any, Optional, Callable, Iterable, AsyncIterator
```

Ruff (0.11.9)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)